### DISCUSSION TITLE
joke languages

### DISCUSSION THREAD

**Discussion by dexter-xyz** [DexterHuang/CyberCodeOnline#2829] [Ideas]
are joke languages applicable? 

such as:
- LOLCAT
- Pirate Speak
- Anglish
- Reverse English
- Upside down English
- Shakespearen English
- 13375P34K
- Furry OwO
- AAAAAAAAAAAAAAA
- Programmer Language
- Google Translated 1000x
- Alien
- Emoji
- Morse Code

these can be separated via divider and located below all languages in the language setting option

**Comment by dunkerquexD** [comment]
Furry OwO
